tools/generator/Tests/Unit-Tests/CodeGeneratorExpectedResults/JavaInterop1/WriteStaticInterfaceProperty.txt

```diff
@@ -0,0 +1,63 @@
+// Metadata.xml XPath interface reference: path="/api/package[@name='java.code']/interface[@name='IMyInterface']"
+[Register ("java/code/IMyInterface", "", "java.code.IMyInterfaceInvoker")]
+public partial interface IMyInterface : IJavaObject, IJavaPeerable {
+	static new readonly JniPeerMembers _members = new JniPeerMembers ("java/code/IMyInterface", typeof (IMyInterface), isInterface: true);
+
+	static Delegate cb_get_Value;
+#pragma warning disable 0169
+	static Delegate Getget_ValueHandler ()
+	{
+		if (cb_get_Value == null)
+			cb_get_Value = JNINativeWrapper.CreateDelegate ((Func<IntPtr, IntPtr, int>) n_get_Value);
+		return cb_get_Value;
+	}
+
+	static int n_get_Value (IntPtr jnienv, IntPtr native__this)
+	{
+		java.code.IMyInterface __this = global::Java.Lang.Object.GetObject<java.code.IMyInterface> (jnienv, native__this, JniHandleOwnership.DoNotTransfer);
+		return __this.Value;
+	}
+#pragma warning restore 0169
+
+	static Delegate cb_set_Value_I;
+#pragma warning disable 0169
+	static Delegate Getset_Value_IHandler ()
+	{
+		if (cb_set_Value_I == null)
+			cb_set_Value_I = JNINativeWrapper.CreateDelegate ((Action<IntPtr, IntPtr, int>) n_set_Value_I);
+		return cb_set_Value_I;
+	}
+
+	static void n_set_Value_I (IntPtr jnienv, IntPtr native__this, int value)
+	{
+		java.code.IMyInterface __this = global::Java.Lang.Object.GetObject<java.code.IMyInterface> (jnienv, native__this, JniHandleOwnership.DoNotTransfer);
+		__this.Value = value;
+	}
+#pragma warning restore 0169
+
+	 static unsafe int Value {
+		// Metadata.xml XPath method reference: path="/api/package[@name='java.code']/interface[@name='IMyInterface']/method[@name='get_Value' and count(parameter)=0]"
+		[Register ("get_Value", "()I", "Getget_ValueHandler")]
+		get {
+			const string __id = "get_Value.()I";
+			try {
+				var __rm = _members.StaticMethods.InvokeInt32Method (__id, null);
+				return __rm;
+			} finally {
+			}
+		}
+		// Metadata.xml XPath method reference: path="/api/package[@name='java.code']/interface[@name='IMyInterface']/method[@name='set_Value' and count(parameter)=1 and parameter[1][@type='int']]"
+		[Register ("set_Value", "(I)V", "Getset_Value_IHandler")]
```

Comment:
It looks like `methods` do this correctly and use `""` if the method is not `virtual`.  Properties do not have this same logic.  Since this is unrelated to DIM I will fix this is a separate PR.